jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -195,6 +187,28 @@ protected void doStart() throws Exception {
     }
 
     super.doStart();
+
+    // Look for a datastore service within the context
+    ClassLoader orig = Thread.currentThread().getContextClassLoader();
```

Comment:
Is that, reflection, to access AE APIs (such as datastore) really necessary (as oppose to specifying AE API jar as "provided" in maven)?